Makefile

```diff
@@ -0,0 +1,68 @@
+VERSION := 0.16.5
```

Comment:
Somewhat confusingly, the library and CLI have different semvers.  This should track the library's semver, which is currently [0.6.3](https://github.com/tree-sitter/tree-sitter/blob/1635aab801152b8b36975dbc6cd02ea01ddbf778/lib/Cargo.toml#L4).

Reply:
That is a perfectly reasonable point to make. This version number is currently stored in a rust binding configuration file, though, which is why I didn't find it. :D

I expected a bit of fine-tuning -- I guessed at what I hoped were the right values, but I guess I was off.

Reply:
Implemented.